debian/neo-go.dirs

```diff
@@ -0,0 +1,2 @@
+var/lib/neo-go
+etc/neo-go
```

Comment:
We need to fix `cli/options/options.go`, move `configPath` into `pkg/config` maybe and set it to `/etc/neo-go` for packaged neo-go.

Reply:
Will it be more suitable if moved to /etc/neofs/neo-go and /var/lib/neofs/neo-go accordingly?
both directories are defined in Unit file anyways, like:
...
ExecStart=/usr/bin/neo-go node --mainnet --config-path=/etc/neofs/neo-go/
...
WorkingDirectory=/var/lib/neofs/neo-go

Reply:
I don't think so, NeoGo is frequently used as a standalone package without NeoFS, so `/etc/neo-go` and `/var/lib/neo-go` are correct directories for it.